Logging_Enabled_installation.md

```diff
@@ -0,0 +1,205 @@
+  resources:
+    requests:
+      storage: 1Gi
+EOF
+// Apply the above PVC
+oc apply -f pvc.yaml
+```
+
+### Configure Results API server to use PVC
+
+```
+cat <<EOF > deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: "1"
+  labels:
+    app.kubernetes.io/name: tekton-results-api
+    app.kubernetes.io/part-of: tekton-results
+    app.kubernetes.io/version: 9f84a1f
+  name: tekton-results-api
+  namespace: tekton-pipelines
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: tekton-results-api
+      app.kubernetes.io/version: 9f84a1f
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/name: tekton-results-api
+        app.kubernetes.io/version: 9f84a1f
+    spec:
+      containers:
+      - env:
+        - name: DB_HOST
+          value: tekton-results-postgres-service.tekton-pipelines.svc.cluster.local
+        - name: DB_USER
+          valueFrom:
+            secretKeyRef:
+              key: POSTGRES_USER
+              name: tekton-results-postgres
+        - name: DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: POSTGRES_PASSWORD
+              name: tekton-results-postgres
+        - name: DB_NAME
+          value: tekton-results
+        image: gcr.io/tekton-releases/github.com/tektoncd/results/cmd/api:9f84a1f@sha256:606816e51ebecb58fccc28f5a95699255ed8742470df673294ce25f69ffc451c
+        imagePullPolicy: IfNotPresent
+        name: api
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            add:
+            - NET_BIND_SERVICE
+            drop:
+            - ALL
+          runAsNonRoot: true
+          seccompProfile:
+            type: RuntimeDefault
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /etc/tekton/results
+          name: config
+          readOnly: true
+        - mountPath: /etc/tls
+          name: tls
+          readOnly: true
+        - name: tekton-logs
+          mountPath: "/logs"
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      serviceAccount: tekton-results-api
+      serviceAccountName: tekton-results-api
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - configMap:
+          defaultMode: 420
+          name: tekton-results-api-config
+        name: config
+      - name: tls
+        secret:
+          defaultMode: 420
+          secretName: tekton-results-tls
+      - name: tekton-logs
+        persistentVolumeClaim:
+            claimName: tekton-logs
+EOF
```

Comment:
Would it be better if we put a kustomize patch (or kubernetes patch) instead of the full YAML.